source/OutfitterPanel.cpp

```diff
@@ -438,7 +445,8 @@ void OutfitterPanel::Buy(bool alreadyOwned)
 			else
 			{
 				// Check if the outfit is for sale or in stock so that we can actually buy it.
-				if(!outfitter.Has(selectedOutfit) && player.Stock(selectedOutfit) <= 0)
+				if(!planet->Outfitter().Has(selectedOutfit) || !CustomSaleManager::CanBuy(*selectedOutfit) ||
+						player.Stock(selectedOutfit) <= 0)
```

Comment:
Why `planet->Outfitter()` now? Can't this just be `outfitter`? You've also changed the meaning of this if statement. Because it was "if the outfitter doesn't have the outfit and it isn't in the player's stock." Now it's "if the outfitter doesn't have it OR the custom sale manager says you can't buy it OR it isn't in the player's stock."

Reply:
Woops
Sometimes I get cpnfused with booleans